pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -9,7 +9,7 @@ enterprise: false
 ---
 
 <p class="message--warning"><strong>WARNING: </strong>
-You can deploy Kaptain to a cluster in a selected Workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the Workspace you are deploying to or move that cluster to another Workspace.
+You can deploy Kaptain to a cluster in a selected workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the workspace you are deploying to or move that cluster to another workspace.
 </p>
```

Comment:
This is one of those things we'll need to clean up eventually - we're capitalizing Workspace in some locations. We'll want the product team to weigh in on this, I suppose.